examples/generate-and-verify-signature/README.md

```diff
@@ -0,0 +1,31 @@
+# Generating and verify a signature
```

Comment:
Write either "Generating and verifying a signature" or "Generate and verify a signature" ...